src/Backoffice/Tasks/App/Request/UpsertTaskRequest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Tasks\App\Request;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Lightit\Backoffice\Tasks\Domain\DataTransferObjects\TaskDto;
+
+class UpsertTaskRequest extends FormRequest
+{
+    public const TITLE = 'title';
+
+    public const DESCRIPTION = 'description';
+
+    public const STATUS = 'status';
+    
+    public const EMPLOYEE_ID = 'employee_id';
+
+    /**
+     * @return array<string, mixed>
+     */
+    public function rules(): array
+    {
+        return [
+            self::TITLE => ['required'],
+            self::DESCRIPTION => ['required'],
```

Comment:
Add string restriction here. Same in the other Request:

```suggestion
            self::TITLE => ['required', 'string'],
            self::DESCRIPTION => ['required', 'string'],
```